solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -85,12 +114,31 @@ const LiquidityRewardCard = ({
           <h6>% of total pool</h6>
         </div>
       </div>
-      <div className={"text-smaller text-grey-70 text-center mb-2"}>
-        * APY calculated assuming $1M pool
-      </div>
+      {!gt(wrappedTokenBalance, 0) && (
+        <div className={"liquidity__new-user-info"}>
+          <Icons.Rewards className={"liquidity__rewards-icon"} />
+          <div className={"liquidity__new-user-info-text"}>
+            <h4>Start earning rewards</h4>
+            <span>
+              Get LP tokens by adding liquidity first to the&nbsp;
+              <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href={viewPoolLink}
+                className="text-white text-link"
+              >
+                Uniswap pool
+              </a>
+            </span>
+          </div>
+        </div>
+      )}
       <div className={"liquidity__token-balance"}>
-        <span className={"liquidity__token-balance_title text-grey-70"}>
-          Reward
+        <h4 className={"liquidity__token-balance__title text-grey-70"}>
```

Comment:
Thoughts about `liquidity__reward-balance` instead of `token-balance`? We display the rewards balance here and I think `token-balance` could be misleading on the CSS side.